cli/bpmetadata/cmd.go

```diff
@@ -47,27 +51,53 @@ func generate(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("error getting working dir: %w", err)
 	}
 
+	//read existing metadata.yaml, if present
+	bpObj := &BlueprintMetadata{}
+	f, _ := os.ReadFile(metadataFileName)
+	// if file is present, try to unmarshal it in BlueprintMetadata
+	if f != nil {
+		bpObj, _ = unmarshalMetadata(f)
+	}
+
+	// if file was present, check if unmarshaling was successful
+	if bpObj == nil && !mdFlags.quiet {
```

Comment:
I think quiet for this usecase is unnecessary since we expect the format to be always valid. If not valid we will exit early rather than do any processing.

Reply:
See comment above

Reply:
Added a `force` flag to generate metadata even if the existing file is erroneous.